app/routes/goal_routes.py

```diff
@@ -1 +1,75 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response
+from app.models.goal import Goal
+from app.models.task import Task
+from ..db import db
+from .helper_methods import validate_model, helper_model_from_dict, helper_get_sorted_query
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    return helper_model_from_dict(Goal, request_body, required_fields=["title"])
+
+@bp.post("/<goal_id>/tasks")
+def add_tasks_to_goal_id(goal_id):
+    goal = validate_model(Goal, goal_id)
+    request_body = request.get_json()
+    
+    task_ids = request_body.get("task_ids", [])
+    # goal.tasks.clear()
+    for task in goal.tasks:
+        task.goal_id = None
+    
+    for id in task_ids:
+        task = validate_model(Task, id)
+        task.goal_id = goal.id
+        # goal.tasks.append(task)
+
+    db.session.commit()
+
+    return {
+        "id": int(goal.id),
+        "task_ids": task_ids
+    }, 200
```

Comment:
Recall that Flask will return a default status code of 200 so we do not need to explicitly return it ourselves. Notice that you don't return an explicit status code in your `get_goals` route on line 44. 

I'd prefer not to explicitly return a status code if I don't need to because it adds more code and feels redundant because the framework does it for us. If you choose to return 200 or choose not to, just be consistent in the choice and apply the decision throughout the entire codebase, instead of sometimes returning it and sometimes not.

```suggestion
    }
```